website/app/components/doc/page/header/search/index.hbs

```diff
@@ -0,0 +1,6 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="doc-algolia-search__autocomplete-container" {{did-insert this.didInsertSearchContainer}}></div>
```

Comment:
This is just a container in which Algolia Autocomplete injects the "trigger" for the search modal (DOM is built by the library, and has very limited possible customization).

We add all our logic via the `didInsertSearchContainer` function invoked once the element is inserted in the DOM.